examples/react-apollo/cypress/support/e2e.ts

```diff
@@ -0,0 +1,33 @@
+import { faker } from '@faker-js/faker'
+
+import 'cypress-mailhog'
+
+declare global {
+  namespace Cypress {
+    interface Chainable {
+      signUpEmailPassword(email: string, password: string): Chainable<Element>
+      confirmEmail(email: string): Chainable<Element>
+    }
+  }
+}
+
+Cypress.Commands.add('signUpEmailPassword', (email, password) => {
+  cy.visit('/sign-up')
+  cy.contains('Continue with email + password').click()
+  cy.get('[placeholder="First name"]').type(faker.name.firstName())
```

Comment:
I think we should incorporate https://github.com/testing-library/cypress-testing-library. Forgot to mention this library when we were discussing the approaches we will take.

Using this library would make our queries a lot more declarative:
```ts
cy.get('[placeholder="Last name"]')
```
would become
```ts
cy.findByPlaceholderText('Last name')
```

There are tons of useful [queries](https://testing-library.com/docs/queries/about) that we could use in the future.

Reply:
nice